cmd/cosign/cli/sign.go

```diff
@@ -223,12 +226,13 @@ func SignCmd(ctx context.Context, keyPath string,
 			}
 		}
 	}
-	index, err := cosign.UploadTLog(signature, payload, pemBytes)
+	inclusionProof, uuid, err := cosign.UploadTLog(signature, payload, pemBytes)
```

Comment:
i could be totally wrong here but I this this needs to be signed by rekor and it's currently not

i had to walk myself through this:
why does it need to be signed if the merkle tree hashes prove inclusion? cause maybe someone lied about the root hash and is giving a fake inclusion proof into a garbage merkle tree.

i think this needs to be signed by rekor... head hurts :P

Reply:
i think you're right! just did a quick skim through rekor code & i think we'd have to add that in (but i might have missed it if it's already there)